src/storage-types/src/time_dependence.rs

```diff
@@ -54,6 +60,42 @@ impl TimeDependence {
         }
     }
 
+    /// Merge any number of dependencies into one, using the supplied refresh schedule.
+    ///
+    /// It applies the following rules under the assumption that the frontier of an object ticks
+    /// at the rate of the slowest dependency. For objects depending on wall-clock time, this is
```

Comment:
I don't think that assumption is correct? For example, a `REFRESH EVERY` MV that depends on a table will tick at the rate of the refresh schedule, not wallclock time.

I see that the logic already existed before this PR, and it made sense to me when I reviewed it initially, so not sure if my mind has advanced or regressed 😅

Reply:
Changing to at the rate of the slowest _immediate_ dependency. Otherwise it's indeed incorrect.

Reply:
I think the comment is still incorrect, given that it says "the frontier of an object ticks". In the `REFRESH EVERY` MV example, the frontier of the MV doesn't tick with wallclock time, even though its dependency does. I think you mean to say something like "ignoring the refresh schedule, the frontier of an object ticks at the rate of the slowest dependency". And the code always retains the refresh schedule separately, which is how we can justify ignoring it when looking at the dependencies.

AFAICT the code does the correct thing, so this is only a documentation issue.